NOTICE

```diff
@@ -9,3 +9,161 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 
 See the Agreement for the specific language governing permissions and
 limitations under the Agreement.
+
+Below is a list of third party open source software which is provided by
+Mesosphere in connection with Mesosphere DC/OS.
+
+dcos-diagnostics
+- Apache License, Version 2.0
+- https://github.com/dcos/dcos-diagnostics/blob/master/LICENSE
+
+adminrouter
```

Comment:
Admin Router belongs to Mesosphere/is a Mesosphere project. What I think we should mention here is OpenSource version of Nginx with its licence: http://nginx.org/LICENSE